api/db/populate-comments.js

```diff
@@ -35,16 +36,14 @@ const prerequisites = (eventsCount, attendantsCount) => {
 
 const getParentComment = (comments) => {
   const parentComments = comments.filter(
-    (comment) => comment.parentComment === undefined
+    (comment) =>
+      comment.parentComment === undefined || comment.parentComment === null
   );
   return parentComments[Math.floor(Math.random() * parentComments.length)];
 };
 
 const createComment = async (commentParams) => {
   const comment = new Comment(commentParams);
```

Comment:
This is where the error above came from: before this line, we should check if `commentParams` has a `parentComment` value of `null` and, if yes, remove it from the object before passing it to the comment constructor. Another possible solution, probably more clear, would be to, on the `createComments` function, only add the `parentComment` value to the `commentParams` object if it's not `null`. If you agree with it, let's go with this second approach, which by the way would be moving the few lines of code that we just deleted below this to the appropriate place and with the appropriate names to the `createComments` function

Reply:
The latter is how it was, but we had decided to simplify it so we could so we could just pass the params to the newly created document (eg. `new Comment(commentParams)` rather than `new Comment({...})`). 

Personally, I believe it is better to include those fields and to leave it as null if it is irrelevant. It will make handling the object much easier as well as providing better structure for the documents (even if that means that the value of a field is null).

Reply:
Yes, I know, that's why we now have the problem again, so the proposed solution is to handle it during the creation of the `commentParams` object instead of here. I would say that we should go that way for now and then we can decide if we want to allow documents to have null values